README.md

```diff
@@ -1,2 +1,161 @@
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse_first_project <a name="about-project"></a>
+   This is my new project I created it using HTML and CSS
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+ - Added index.html file.
+ - Added style.css file.
+ - Added check errors with linters 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. web browser.
+2. Code editor.
+3. Git-smc 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/sadaf-Daneshgar/Microverse_first_project.git
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd my-project
+  npm install
+```
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run tests
+```sh
+  npm init -y
+  npm install --save-dev hint@7.x
+```
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Sadaf Daneshgar
+
+- GitHub: [@githubhandle](https://github.com/sadaf-Daneshgar)
+- Twitter: [@twitterhandle](https://twitter.com/SadafDaneshgar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sadaf-barekzai-00480a242)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] Add Header.
+- [ ] Add Style.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project just give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The link to your LICENSE is broken and displays an error page. I would advise you to create a valid LICENSE for this project. To create a valid LICENSE, kindly follow the instructions at the top-right part of this article https://choosealicense.com/licenses/mit/. 👍 😎